ts/Core/Axis/Axis.ts

```diff
@@ -4783,7 +4783,7 @@ class Axis {
         var pos,
             lastPos,
             roundedMin =
-                correctFloat(Math.floor(min / tickInterval) * tickInterval),
+                Math.max(correctFloat(Math.floor(min / tickInterval) * tickInterval), -Number.MAX_VALUE),
```

Comment:
This `Math.max` doesn't seem to make a difference, as far as I can see this is handled in the new `clamp`?

Reply:
The first tick position would be -Infinity without it.